impeller/entity/contents/filters/filter_contents.h

```diff
@@ -178,11 +184,28 @@ class FilterContents : public Contents {
   virtual void SetRenderingMode(Entity::RenderingMode rendering_mode);
 
  private:
+  /// @brief  Internal utility method for |GetLocalCoverage|.
+  ///
+  ///         |GetLocalCoverage| will intersect the filter's coverage as
+  ///         computed by this method with the oject's coverage hint.
+  ///         This method should compute the coverage of the associated
+  ///         inputs and then apply any modifications to that coverage
+  ///         representing the actions of the filtering operation, ignoring
+  ///         the coverage hint.
   virtual std::optional<Rect> GetFilterCoverage(
       const FilterInput::Vector& inputs,
       const Entity& entity,
       const Matrix& effect_transform) const;
 
+  /// @brief  Internal utility method for |GetSourceCoverage|.
```

Comment:
This one is mostly good, but make the brief define what the function does briefly.  Also define effect_transform.  You'll probably just want to link to a definition somewhere else in this file.

What is the coordinate space of the `output_limit`?

Reply:
`|GetSourceCoverage|` is a link to the full definition...